mods/stairs/init.lua

```diff
@@ -256,355 +373,425 @@ if replace then
 end
 
 
--- Stair/slab registration function.
--- Nodes will be called stairs:{stair,slab}_<subname>
+-- Stair/slab/stairi/stairo registration function.
+-- Nodes will be called stairs:{stair,slab,stairi,stairo}_<subname>
 
-function stairs.register_stair_and_slab(subname, recipeitem,
-		groups, images, desc_stair, desc_slab, sounds)
+function stairs.register_stair_and_slab_and_stairi_and_stairo(subname, recipeitem,
```

Comment:
That is an incredibly long function. Maybe this could be shortened to something simple like `stairs.register`?